protocol-definitions/custom/Custom.yaml

```diff
@@ -1499,3 +1499,15 @@ customTypes:
         type: String
         since: 2.7
         nullable: true
+  - name: Version
+    since: 2.7
```

Comment:
```suggestion
    since: 2.8
```
For 5.5 release, we need to increment the version.

Reply:
74593a07